pyds9/setup_package.py

```diff
@@ -138,6 +139,12 @@ def post_build_ext_hook(cmd):
         print(" ".join(compile_cmd))
         sp.check_call(compile_cmd)
 
+    # run make clean
+    xpa_dir = [i for i in libxpa.include_dirs if 'xpa' in i][0]
+
+    with cd(xpa_dir):
+        sp.check_call(['make', 'distclean'])
```

Comment:
@ericmandel : I do this to cleanup the pyds9 repo after running ``configure``. ``distclean`` removes all the temporary files except ``conf.h`` and ``xpa.pc``.
Should ``distclean`` clean also them?
If there are good reason for this behaviour I can either:
* remove them after making distclean
* patch the local makefile/configure to remove also them
* add them to gitignore